src/autocomplete.js

```diff
@@ -274,16 +291,16 @@ class Autocomplete extends Component {
       () => {
         // call the provided callback if it's a callback
         cbToCb(cb)()
-        // if the selectedValue changed
```

Comment:
As I was working with things it seemed to me to be a bit better to have this happen after `onStateChange` because `onStateChange` happens _all_ the time, so I felt it better to have `onChange` win (because it comes second) in the event of a state change conflict for the user.